Cargo.toml

```diff
@@ -15,6 +15,10 @@ exclude = [
     "fluent-cli",
 ]
 
+[workspace.package]
+edition = "2021"
+rust-version = "1.67.0"
```

Comment:
Where is this status quo from?

Reply:
From bisecting builds until I figured out what this can actually be built with, using `cargo msrv`.